feature-libs/product-configurator/rulebased/occ/cpq/cpq-configurator-occ.module.ts

```diff
@@ -0,0 +1,30 @@
+import { CommonModule } from '@angular/common';
+import { NgModule } from '@angular/core';
+import { ConfigModule } from '@spartacus/core';
+import { OccConfiguratorCpqUpdateCartEntrySerializer } from './converters';
+import {
+  CPQ_CONFIGURATOR_ADD_TO_CART_SERIALIZER,
+  CPQ_CONFIGURATOR_UPDATE_CART_ENTRY_SERIALIZER,
+} from './converters/cpq-configurator-occ.converters';
+import { OccConfiguratorCpqAddToCartSerializer } from './converters/occ-configurator-cpq-add-to-cart-serializer';
+import { defaultOccCpqConfiguratorConfigFactory } from './default-occ-configurator-cpq-config';
+
+@NgModule({
+  imports: [
+    CommonModule,
+    ConfigModule.withConfigFactory(defaultOccCpqConfiguratorConfigFactory),
```

Comment:
Please use `provideDefaultConfig()` in `providers` array. More details in https://github.com/SAP/spartacus/issues/11850

[nitpick] Btw. no need for factory.

Reply:
Pick!

Reply:
Done!